ckan/model/types.py

```diff
@@ -43,6 +43,8 @@ class JsonType(types.TypeDecorator):  # type: ignore
     '''
     impl = types.UnicodeText
 
+    cache_ok = False
```

Comment:
this seems like a separate bug-fix. Are we working around this issue in other parts of our code, like in the handling of plugin_extras fields?

Reply:
It's not exactly a bug fix, I'd say. That's the default behavior of SQLAlchemy now. But if you don't specify `cache_ok` explicitly, SQLAlchemy will produce warnings(and errors in the future) and set this attribute to False.

This JsonType/JsonDictType is used only by activity.data, resource.extras, and resource_view.config columns. 
For other cases, like `plugin_extras` of user\package, we are using native JSON/JSONB fields. And it would be nice to remove this class completely and switch to the native type here as well, but it's a bigger change, that's why I only explicitly set `cache_ok = False` to suppress errors.